### PR TITLE
added mysql to docker and connected with django

### PR DESCRIPTION
**Pull request by yt**
Set up mysql local inside docker and connected it to Django

### CODE REVIEW

File: webapp/settings.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```
Comment (ugursogukpinar) [inline webapp/settings.py:28]: @yt It should be configurable via .env

File: docker-compose.yml [hunk 2]
```diff
@@ -2,10 +2,18 @@ version: '3'
 
 services:
 
+  database:
+    image: mysql:5.6
+    environment:
+      MYSQL_DATABASE: mysql_db
```
Comment (ugursogukpinar) [inline docker-compose.yml]: @yt Let's use a proper name for database.

Comment (yt) [reply docker-compose.yml]: @ugursogukpinar changed it to `news_feed_db`

File: webapp/settings.py (hunk 1 above)
Comment (yt) [reply webapp/settings.py:28]: It is, at the end of settings file if environment is local, the settings gets overridden by settings_development.py file.

**Review by ugursogukpinar — APPROVED** [review]

**Pull request merged**